qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
Right, was getting sysmessage and message variables confused. Point still stands though: we do need to disable this in maintenance mode, as we don't want people trying to make database changes while in maintenance mode.

Could we also use `level == success` in this, since that implies it successfully did something? I'm trying to have as few new variables added to the templates as possible since it makes it more complicated to edit them down the line.

As for tests use the tornado testing base I made and set the redis key/send erroneous post data to make sure the pages render properly. Take a look at the other tests for how to do this.